stack/indexer/indexer-security-init.sh

```diff
@@ -42,11 +42,17 @@ fi
 
 # -----------------------------------------------------------------------------
 
+eval "wazuh_indexer_root_ca=( $(cat /etc/wazuh-indexer/opensearch.yml | grep plugins.security.ssl.transport.pemtrustedcas_filepath | sed 's/plugins.security.ssl.transport.pemtrustedcas_filepath:\s//') )"
+WAZUH_INDEXER_ADMIN_PATH="${wazuh_indexer_root_ca%%/root-ca.*}"
 securityadmin() {
 SECURITY_PATH="${INSTALL_PATH}/plugins/opensearch-security"
 
-OPENSEARCH_PATH_CONF="${CONFIG_PATH}" JAVA_HOME="${INSTALL_PATH}/jdk" runuser wazuh-indexer --shell="/bin/bash" --command="${SECURITY_PATH}/tools/securityadmin.sh -cd ${SECURITY_PATH}/securityconfig -cacert ${CONFIG_PATH}/certs/root-ca.pem -cert ${CONFIG_PATH}/certs/admin.pem -key ${CONFIG_PATH}/certs/admin-key.pem -h ${HOST} -p ${PORT} ${OPTIONS}"
-
+if [ -f "${WAZUH_INDEXER_ADMIN_PATH}/admin.pem" ] && [ -f "${WAZUH_INDEXER_ADMIN_PATH}/admin-key.pem" ]; then
```

Comment:
Check also if root CA is present.

Reply:
Done! https://github.com/wazuh/wazuh-packages/commit/80727a55d4c3d650f180e3c67492d579c147a5f8